File: Miscellaneous/Python/04 Fibonacci Series.py
#Write a Python program to print the Fibonacci sequence up to a given number of terms.
def fibonacci(n):
    num=[0,1]
    while len(num)<n:
        num.append(num[-1]+num[-2])
    return num
a=int(input("Enter the number of series to be added in the fibonacci series: "))
print(fibonacci(a))

File: Miscellaneous/CPP/04 Fibonacci Series.cpp
#include<iostream>
using namespace std;
void fibonacci(int n){
  int first = 0;
  int second = 1;
  cout << first << " " << second << " ";
  for(int i = 2; i < n; i++){
    int next = first + second;
    first = second;
    second = next;
    cout << next << " ";
  }
}
int main(){
  int num;
  cout << "Enter the number of terms: ";
  cin >> num;
  fibonacci(num);
  return 0;
}